ci/before_deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+root=$(realpath "$(dirname "$0")/..")
+version=$(TZ=UTC date +v%Y%m%d)
+name=cquery-$version-x86_64-unknown-linux-gnu
+pkg=$(mktemp -d)
+
+cd "$root/build/release"
+rsync -rtLR bin ./lib/clang+llvm-*/lib/libclang.so.? ./lib/clang+llvm-*/lib/clang/*/include "$pkg/"
+
+cd "$pkg"
+strip -s bin/cquery lib/clang+llvm*/lib/libclang.so.?
+if [[ $(uname) == Linux ]]; then
+  # ./bin/cquery -> $name/bin/cquery
+  tar -Jcf "$root/build/$name.tar.xz" --owner 0 --group 0 --xform "s,^\./,$name/," .
```

Comment:
tar.xz does not extract easily on mac without additional software. We should publish a different format. http://www.cs.cmu.edu/~dmortens/xz.html

Reply:
Changed to .tar.gz